assignments/prototypes.js

```diff
@@ -32,16 +45,33 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
- 
+function Humanoid(attribute) {
+  this.team = attribute.team;
+  this.weapons = attribute.weapons;
+  this.language = attribute.language;
+  this.greet = function () {
+    return `${this.name} offers a greeting in ${this.language}.`
+  }
+
+}
```

Comment:
And this is not set up right either:

function Humanoid(attributes) {
  CharacterStats.call(this, attributes);
  this.faction = attributes.faction;
  this.weapons = attributes.weapons;
  this.language = attributes.language;
}

Humanoid.prototype = Object.create(CharacterStats.prototype);  // inherits takeDamage, as well as destroy from above

Humanoid.prototype.greet = function() {
  return `${this.name} offers a greeting in ${this.language}.`;  // adds a greet function to the Humanoid prototype
}